pycon/static/less/site.less

```diff
@@ -391,6 +391,7 @@ aside {
         -moz-transform: translate(-50%,0);
         -ms-transform: translate(-50%,0);
```

Comment:
So these transforms are exactly what's causing this. If you remove them and then decrease `margin-left` to `5%` the site looks the same and the problem disappears. Please remove the negative `margin-right`.

To be clear, this works on Chrome and Firefox. I haven't tested Safari, IE, or any of the mobile browsers.